### PR TITLE
KOA-5577 Update theme colours

### PR DESCRIPTION
**Pull request by marianeum**
Also cleaned up the gradient component and any other deprecated colour usages

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-import androidx.test.rule.ActivityTestRule
-import net.skyscanner.backpack.test.R
-import org.junit.Assert
-import org.junit.Before
-import org.junit.Rule
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4::class)
-class BpkThemeTest {
```
Comment (marianeum) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/BpkThemeTest.kt:35]: we were just testing a deprecated function and default theming functionality here

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/ResourcesUtilTest.kt [hunk 2]
```diff
@@ -1,54 +0,0 @@
...
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import androidx.test.platform.app.InstrumentationRegistry
-import net.skyscanner.backpack.R
-import org.junit.Assert
-import org.junit.Before
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4::class)
-class ResourcesUtilTest {
```
Comment (marianeum) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/ResourcesUtilTest.kt:32]: again - this is just testing how android works?!

File: Backpack/src/main/res/color/bpk_chip_background_color.xml [hunk 3]
```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (marianeum) [inline Backpack/src/main/res/color/bpk_chip_background_color.xml:1]: these were unused

File: Backpack/src/main/res/drawable-nodpi/bpk_self_service.xml [hunk 4]
```diff
@@ -1,3 +0,0 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
```
Comment (marianeum) [inline Backpack/src/main/res/drawable-nodpi/bpk_self_service.xml:1]: this was somehow duplicated in another package and therefore overriding the original icon

File: Backpack/src/main/res/layout/view_bpk_calendar.xml [hunk 5]
```diff
@@ -29,7 +29,7 @@
     android:layout_marginTop="@dimen/bpkSpacingBase"
     android:visibility="gone"
     android:layout_centerHorizontal="true"
-    app:badgeType="dark"
```
Comment (marianeum) [inline Backpack/src/main/res/layout/view_bpk_calendar.xml:32]: dark is deprecated - should be strong based on design

File: Backpack/src/main/res/values/style.xml [hunk 6]
```diff
@@ -26,7 +26,7 @@
     <item name="android:layout_weight">1</item>
     <item name="android:lines">1</item>
     <item name="android:ellipsize">end</item>
-    <item name="android:textColor">@color/bpkTextQuaternary</item>
```
Comment (marianeum) [inline Backpack/src/main/res/values/style.xml:29]: this was missed in the calendar update

File: Backpack/src/main/res/layout/view_bpk_calendar_2.xml [hunk 7]
```diff
@@ -34,7 +34,7 @@
     android:layout_height="wrap_content"
     android:layout_marginTop="@dimen/bpkSpacingBase"
     android:visibility="gone"
-    app:badgeType="dark"
```
Comment (frugoman) [inline Backpack/src/main/res/layout/view_bpk_calendar_2.xml:37]: 👍

**Review by frugoman — APPROVED** [review]

**Pull request merged**